### PR TITLE
IGAPP-1128: Replaced deprecated keyCode event property

### PR DESCRIPTION
**Pull request by Cmd190**
KeyboardEvent.keyCode was replaced.

This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]
Nice!

File: web/src/hooks/useOnClickOutside.ts [hunk 1]
```diff
@@ -12,11 +12,11 @@ const useOnClickOutside = (ref: { current: HTMLElement | null }, callback: () =>
     }
 
     const handleKeyPress = (event: KeyboardEvent) => {
-      const KEY_ENTER = 13
-      const KEY_ESCAPE = 27
-      if (event.keyCode === KEY_ESCAPE) {
+      const KEY_ENTER = 'Enter'
```
Comment (SteffiStuffel) [inline web/src/hooks/useOnClickOutside.ts:15]: I think now these constants are not needed any longer, as the naming is already clear.

**Review by steffenkleinle — COMMENTED** [review]
Tested on firefox. Minor since it probably only affects a rare minority of users like me: I switched Escape and Caps Lock on my keyboard but I still have to use the same keys here, i.e. `Escape` to close instead of `Caps Lock`. Before it worked fine for me to close with `Caps Lock`. Maybe you could investigate if there is a way to fix this?

File: web/src/hooks/useOnClickOutside.ts (hunk 1 above)
Comment (Cmd190) [reply web/src/hooks/useOnClickOutside.ts]: done

**Comment by Cmd190** [comment]
> Tested on firefox. Minor since it probably only affects a rare minority of users like me: I switched Escape and Caps Lock on my keyboard but I still have to use the same keys here, i.e. `Escape` to close instead of `Caps Lock`. Before it worked fine for me to close with `Caps Lock`. Maybe you could investigate if there is a way to fix this?

Yes this was because it checked for the code send from the keyboard and not the actual value. I changed to the `key` event which should do the jobt

**Review by steffenkleinle — APPROVED** [review]
Good stuff!

**Pull request merged**